content/TrueCommand/Clustering/ManageClusters.md

```diff
@@ -8,129 +8,127 @@ weight: 20
 Clustered volumes have differing management options based on the cluster type.
 
 {{< hint danger >}}
-Removing and/or replacing bricks from a clustered volume can lead to data corruption. Do not attempt to use this feature at this time. 
+Removing or replacing bricks from a clustered volume can lead to data corruption. Do not attempt to use this feature at this time. 
 {{< /hint >}}
 
 {{< tabs "Types of Clustered Volumes" >}}
 {{< tab "Distributed" >}}
 
 ![DistributedClusterCard](/images/TrueCommand/2.0/DistributedClusterCard.png "DistributedClusterCard")
 
-**Distributed-Cluster** volume the editing options are, **Add Brick**, **Replace Brick**, and **Delete** the cluster.
+**Distributed-Cluster** volumes have three editing options: **Add Brick**, **Remove Brick**, and **Delete**.
 
 ![DistributedClusterOptions](/images/TrueCommand/2.0/DistributedClusterOptions.png "DistributedClusterOptions")
 
 ### Add a brick to a Distributed Cluster
 
-Click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon on the cluster overview card and select the **+ Add Brick** option to open the **Add Brick** menu.
+Click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon on the cluster overview card and select the **+ Add Brick** option to open the **Add Brick to Distributed-Cluster** menu.
 
 ![DistributedClusterOptionsAdd](/images/TrueCommand/2.0/DistributedClusterOptionsAdd.png "DistributedClusterOptionsAdd")
 
 Click **Brick Choices** to display the list of available systems.
 
 ![DistributedClusterOptionsAddBrick](/images/TrueCommand/2.0/DistributedClusterOptionsAddBrick.png "DistributedClusterOptionsAddBrick")
 
-Selecting a system displays options for the created brick. iXsystems strongly recommends that you match the size of the existing bricks but you can change this if required.
+Selecting a system displays options for the brick. iXsystems strongly recommends that you match the size of the existing bricks, but you can change this if required.
 
 ![DistributedClusterOptionsAddBrickOptions](/images/TrueCommand/2.0/DistributedClusterOptionsAddBrickOptions.png "DistributedClusterOptionsAddBrickOptions")
 
-When you are satisfied with the settings, click **SAVE** to add the brick. TrueCommmand and TrueNAS SCALE adds the brick to the cluster.
+Once you are satisfied with the settings, click **SAVE** to add the brick to the cluster.
```

Comment:
```suggestion maybe try after instead of once as you do this after and not just a single time
After you are satisfied with the settings, click **SAVE** to add the brick to the cluster.
```